.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: node_js
+node_js:
+  - "7"
+before_script:
+  - export DISPLAY=:99.0
+  - sh -e /etc/init.d/xvfb start
```

Comment:
xvfb isn't needed anymore with recent Chrome and Firefox versions, you can use the `-headless` flag

Reply:
cool, will try.